transport.go

```diff
@@ -38,13 +40,19 @@ func (t *Transport) RoundTrip(req *http.Request) (resp *http.Response, err error
 	span.SetAttributes(semconv.HTTPTargetKey.String(req.URL.Path))
 	span.SetAttributes(semconv.HTTPHostKey.String(req.URL.Host))
 	t.propagator.Inject(traceCtx, propagation.HeaderCarrier(req.Header))
-
+	// maximum size to lumigoSpans RD-7825
+	maxEntrySize := getenv("DEFAULT_MAX_ENTRY_SIZE", 2048)
```

Comment:
Can you extract it to a function? I think it will be used in more places in the future.

Reply:
Notice that the full name is `LUMIGO_DEFAULT_MAX_ENTRY_SIZE`, `LUMIGO_` is added automatically by viper.